### PR TITLE
fix: subdomainLinks & available connectors 

### LINKED ISSUE #306: No confirmation modal tx on renewal of a domain
Capture d’écran 2023-07-19 à 11 08 22

This button don't have any renewal tx modal

### LINKED ISSUE #307: Web wallet autoconnect issue
Sometimes when I click on Connect the web wallet window opens automatically and I have to reload in order to connect with my Braavos or Argent wallet extension.

Seems that there is some sort of autoConnect of webwallet we have to delete

### LINKED ISSUE #308: Impossible to access xplorer.stark
when I click on .xplorer.stark domains I can't access and make action on it.

We need to have exactly the same behaviour as Braavos subdomains

### PR DESCRIPTION
**Pull request by irisdv**
- refreshes `available` connectors on `address` change : closes #307 
- adds modal tx on renewal of a domain - closes #306 
- fixes links for xplorer subdomains : closes #308

### CODE REVIEW

File: components/identities/identitiesGalleryV1.tsx [hunk 1]
```diff
@@ -67,7 +67,13 @@ const IdentitiesGalleryV1: FunctionComponent<IdentitiesGalleryV1Props> = ({
                 />
               );
             case "xplorer":
-              return <ArgentIcon width={"150px"} color="#f36a3d" />;
+              return (
+                <ArgentIcon
+                  width={"150px"}
+                  color="#f36a3d"
+                  onClick={() => router.push(`/externaldomains/${domain}`)}
```
Comment (fricoben) [inline components/identities/identitiesGalleryV1.tsx]: To not change the `IconProps` type it's better to add the OnClick on a wrapper div imo, what do you think ?

**Review by Th0rgal — APPROVED** [review]
lgtm

**Review by fricoben — CHANGES_REQUESTED** [review]
Smalll change

**Review by fricoben — APPROVED** [review]
Lgtm

**Pull request merged**